common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -402,24 +405,13 @@ class @Problem
             MathJax.Hub.Queue ["Typeset", MathJax.Hub, element]
 
         `// Translators: the word Answer here refers to the answer to a problem the student must solve.`
-        @$('.show-label').text gettext('Hide Answer')
+
         @el.addClass 'showed'
+        $('html, body').animate({
+          scrollTop: @questionTitle.offset().top
+        }, 500);
         @updateProgress response
         window.SR.readElts(answer_text)
```

Comment:
@cptvitamin I'm not sure what this line is. Is there really something that will read aloud the answer_text? How does that interact with the focus switch to the top of the problem (not happening currently, but part of the acceptance criteria).

**window.SR.readElts(answer_text)**